templates/shared/_ubuntu_advantage_server.html

```diff
@@ -2,93 +2,68 @@
   <thead>
     <tr>
       <th scope="col" class="row-title pricing-features-column">Features</th>
-      <th scope="col">Essential</th>
       <th scope="col">Standard</th>
       <th scope="col">Advanced</th>
     </tr>
   </thead>
   <tbody>
-    <!--<tr>
-      <td>&nbsp;</td>
-      <td>Ideal for Ubuntu scale out workloads and microservice architectures  in the enterprise</td>
-      <td>Perfect for Ubuntu in production data centres hosting virtualisation and container</td>
-      <td>Tailored for mission critical Ubuntu clouds and clusters, where uptime and availability are imperative</td>
-  </tr>-->
-    <tr>
-      <th>24x7 self-service customer care portal and knowledge base</th>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-    </tr>
     <tr>
       <th>8x5 ticket support for the default packages in the Ubuntu Server image</th>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
     </tr>
     <tr>
       <th>Landscape Management</th>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
     </tr>
     <tr>
       <th>Ubuntu Legal Assurance programme</th>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
       <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
-      <td><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes" /></td>
     </tr>
     <tr>
       <th>Live Patching (16.04 only)</th>
```

Comment:
You could update this and copy doc to mention `14.04` if you're feeling energetic.